components/screen-layout/ContentWithAvatar.tsx

```diff
@@ -11,6 +12,7 @@ type ContentWithAvatarProps = {
   variant?: 'success' | 'error'
   customAvatarComponent?: ReactNode
   actionButton?: ReactNode
+  markdown?: boolean
```

Comment:
We try to indicate boolean props by proper name, usually using `is` prefix, or verb prefix. It also make sense to use `asSomething` (e.g.`asChild` in `Link` component).
From our Strapi best practices docs: https://bratislava.github.io/strapi/best-practices#field-naming

I would say, `asMarkdown` is okay for this case?

Reply:
a good point 👍 renamed to `asMarkdown`